Tests/PostProcessing_test/main.py

```diff
@@ -0,0 +1,254 @@
+
+def ThreeDTest():
+    """Executes the test routine for the 3D postprocessing module
+
+    Returns:
+        boolean: success status of the test
+    """
+    print("\n\n\n\n")
+
+    # Since this is a test script, we remove previous results
+    if os.path.exists("3d_plots"):
+        print("Removing previous 3d_plots")
+        shutil.rmtree("3d_plots")
+    if os.path.exists("3d_movies"):
+        print("Removing previous 3d_movies")
+        shutil.rmtree("3d_movies")
+
+    # Execute the tests
+    print("Executing 3-D plot test...")
+    subprocess_cmd = ["visit", "-nowin", "-cli", "-s", "../../PostProcessing/VisitPlot.py", "3DPlotParams.ini"]
+    subprocess_failure_status = subprocess.call(subprocess_cmd)
+    if subprocess_failure_status == 0 or subprocess_failure_status == 250:
+        subprocess_failure_status = False
+    else:
+        subprocess_failure_status = True
+    
+    if subprocess_failure_status:
+        print("subprocess status code: ", subprocess_failure_status)
+        print("subprocess command: {0}".format(subprocess_cmd))
+        raise RuntimeError("3-D plot test failed to execute!")
+
+    return not subprocess_failure_status
+    
+
+if __name__ == "__main__": 
+
+    #download test dataset from Zenodo
+    download_dataset() 
+
+    # Check if we're running VisIt
+    try:
+        __visit_script_file__
+        running_visit = True
+    except NameError:
+        running_visit = False
+
+    #if we're not running VisIt, then we're running python.
+    if not running_visit:
+        print("\n\n##############################")
+        print("Running line test with python")
+        print("##############################\n\n")
+        line_success_status = LineTest()
+
+        if line_success_status:
+            print("\n\n###############")
+            print("Tests passed!")
+            print("###############\n\n")
+        else:
+            print("\n\n###############")
+            print("Line test failed!")
+            print("###############\n\n")
+
+    elif running_visit:
+        print("\n\n##############################")
+        print("Running 2D and 3D test with VisIt")
+        print("##############################\n\n")
+
+        #run postprocessing for VisIt modules
+        twod_success_status = TwoDTest()
+        threed_success_status = ThreeDTest()
+
+        success_status = (twod_success_status and threed_success_status)
+
+        if success_status:
+            print("\n\n###############")
+            print("Tests passed!")
+            print("###############\n\n")
+        else:
+            if not twod_success_status:
+                print("\n\n###############")
+                print("2D test failed!")
+                print("###############\n\n")
+            if not threed_success_status:
+                print("\n\n###############")
+                print("3D test failed!")
+                print("###############\n\n")
+
+        sys.exit(0 if success_status else 1)
```

Comment:
```suggestion
        sys.exit(success_status)
```
should probably work